packages/template-retail-react-app/app/components/item-variant/item-attributes.jsx

```diff
@@ -68,6 +113,67 @@ const ItemAttributes = ({includeQuantity, currency, ...props}) => {
                 </Text>
             )}
 
+            {productBundleData && !productBundleVariantData && (
+                <Box>
+                    {productBundleData?.bundledProducts.map(({product, quantity}) => (
+                        <Box marginTop={2} key={product.id}>
+                            <Text fontSize="sm" color="gray.700" as="b">
+                                {product?.name}
+                            </Text>
+                            <Text fontSize="sm" color="gray.700">
+                                {intl.formatMessage({
+                                    defaultMessage: 'Qty',
+                                    id: 'add_to_cart_modal.label.quantity'
+                                })}
+                                : {quantity}
+                            </Text>
+                        </Box>
+                    ))}
+                </Box>
+            )}
+
+            {productBundleVariantData && (
```

Comment:
I think this needs a ternary for isLoading derived from the isLoading on this hooks call
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1304/files#diff-fa05e6157a39878cae017b7d39ce728b839fc2952c8b96b6319cd1c8f9150aacR58

**Also** needs some product `type` checking to determine whether it's of the right type to initiate that follow up API call